mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/ui/Layout.kt

```diff
@@ -37,16 +37,15 @@ internal sealed class NoContentMeasureScope {
 
 @Composable
 @MosaicComposable
-@Suppress("ktlint:compose:param-order-check") // Order is correct, check just can't tell.
```

Comment:
`NoContentMeasurePolicy` is picked up as a lambda as it is defined in this same file (after 0.3.8 of compose-rules). In case this class was moved elsewhere, the spotless config should have this class added to the `compose_treat_as_lambda` list -- but as it is now, it's not necessary.